adapters/elbv2/elb.go

```diff
@@ -295,9 +296,36 @@ func NewLoadBalancerSource(client elbClient, accountID string, region string) *s
 				LoadBalancerArns: []string{query},
 			}, nil
 		},
-		PaginatorBuilder: func(client elbClient, params *elbv2.DescribeLoadBalancersInput) sources.Paginator[*elbv2.DescribeLoadBalancersOutput, *elbv2.Options] {
+		PaginatorBuilder: func(client elbClient, params *elbv2.DescribeLoadBalancersInput) adapters.Paginator[*elbv2.DescribeLoadBalancersOutput, *elbv2.Options] {
 			return elbv2.NewDescribeLoadBalancersPaginator(client, params)
 		},
 		OutputMapper: loadBalancerOutputMapper,
 	}
 }
+
+func LoadBalancerMetadata() sdp.AdapterMetadata {
+	return sdp.AdapterMetadata{
+		Type:            "elbv2-load-balancer",
+		DescriptiveName: "Elastic Load Balancer",
+		SupportedQueryMethods: &sdp.AdapterSupportedQueryMethods{
+			Get:               true,
+			List:              true,
+			Search:            true,
+			GetDescription:    "Get an ELB by name",
+			ListDescription:   "List all ELBs",
+			SearchDescription: "Search for ELBs by ARN",
+		},
+		TerraformMappings: []*sdp.TerraformMapping{
+			{
+				TerraformQueryMap: "aws_lb.arn",
+				TerraformMethod:   sdp.QueryMethod_SEARCH,
+			},
+			{
+				TerraformQueryMap: "aws_lb.id",
+				TerraformMethod:   sdp.QueryMethod_SEARCH,
```

Comment:
```suggestion
				TerraformMethod:   sdp.QueryMethod_GET,
```

Given that GET queries by "Name" I'm assuming this is the correct method for an ID